scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulingService.java

```diff
@@ -349,6 +351,19 @@ public boolean kill() {
 
         listener.schedulerStateUpdated(SchedulerEvent.KILLED);
 
+        // To properly exit the java scheduling process
+        new Thread(new Runnable() {
+            @Override
+            public void run() {
+                try {
+                    Thread.sleep(SCHEDULER_EXIT_DELAY);
+                } catch (InterruptedException e) {
+                    e.printStackTrace();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use a logger to log this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1148)

Reply:
yes, use a logger